src/bot/commands/match_helpers.py

```diff
@@ -0,0 +1,290 @@
+
+    # Escape markdown up to this point
+    output_message = escape_markdown(output_message, version=2)
+
+    opendota_link = f"https://www.opendota.com/matches/{match.match_id}"
+    dotabuff_link = f"https://www.dotabuff.com/matches/{match.match_id}"
+
+    output_message += (
+        f"\n\nMore information: [OpenDota]({opendota_link}), [Dotabuff]({dotabuff_link})"
+    )
+
+    return output_message
+
+
+def build_default_player_string(player):
+    kills = player.kills
+    deaths = player.deaths
+    assists = player.assists
+    kda = f"{kills}/{deaths}/{assists}"
+
+    last_hits = player.last_hits
+    denies = player.denies
+    cs = f"{last_hits}/{denies}"
+
+    level = player.level
+
+    xpm = player.xp_per_min
+    gpm = player.gold_per_min
+
+    hero_data = helpers.get_hero_data(player.hero_id)
+    hero_name = hero_data["localized_name"]
+
+    return f"{kda} | L{level} | {cs} | {gpm} GPM | {xpm} XPM | {hero_name}\n"
+
+
+# Players/rank view
+
+def build_players_match_message(match, player_data):
+    output_message = "\nRadiant:\n"
+    for player in player_data:
+        if player.isRadiant:
+            output_message += build_players_player_line(player)
+    
+    output_message += "\nDire:\n"
+    for player in player_data:
+        if not player.isRadiant:
+            output_message += build_players_player_line(player)
+
+    output_message = escape_markdown(output_message, version=2)
+    return output_message
+
+
+def build_players_player_line(player):
+    try:
+        player_name = player.personaname
+    except:
+        player_name = "Anonymous"
+
+    hero_data = helpers.get_hero_data(player.hero_id)
+    hero_name = hero_data["localized_name"]
+
+    if player.rank_tier:
+        rank = f"| {helpers.map_rank_tier_to_string(player.rank_tier)}"
+    else:
+        rank = ""
+    
+    party_letter = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J"][player.party_id]
+    
+    return f"{party_letter} | {player_name} ({hero_name}) {rank}\n"
+
+
+# Damage and healing done
+
+def build_damage_match_message(match, player_data):
+    output_message = "\nRadiant:\n"
+    for player in player_data:
+        if player.isRadiant:
+            output_message += build_damage_player_line(player)
+    
+    output_message += "\nDire:\n"
+    for player in player_data:
+        if not player.isRadiant:
+            output_message += build_damage_player_line(player)
+
+    output_message = escape_markdown(output_message, version=2)
+    return output_message
+
+
+def build_damage_player_line(player):
+    damage = player.hero_damage
+    building_damage = player.tower_damage
+    healing = player.hero_healing
+
+    hero_data = helpers.get_hero_data(player.hero_id)
+    hero_name = hero_data["localized_name"]
+
+    return f"{damage} DMG | {building_damage} TD | {healing} H | {hero_name}\n"
+
+
+# Pick order
+
+def build_order_match_message(match, player_data):
```

Comment:
This function name could be slightly more descriptive. Such as, `build_pick_ban_order_match_message`